containers/Dockerfile.buildenv

```diff
@@ -134,7 +134,7 @@ FROM --platform=$BUILDPLATFORM ubuntu:22.04 AS xbuild
 WORKDIR /root
 COPY --from=xx / /
 ARG TARGETPLATFORM
-ARG PARALLEL_MAKE_ARG=-j2
+ARG PARALLEL_MAKE_ARG=-j4
```

Comment:
(optional) You can set this variable on CI as well:

```diff
diff --git a/.github/workflows/BuildDockerContainer.yaml b/.github/workflows/BuildDockerContainer.yaml
index 39f0d239b4..85c1d19853 100644
--- a/.github/workflows/BuildDockerContainer.yaml
+++ b/.github/workflows/BuildDockerContainer.yaml
@@ -60,6 +60,7 @@ jobs:
           platforms: >
             ${{ inputs.build_arm && 'linux/amd64,linux/arm64'
             || 'linux/amd64' }}
+          build-args: PARALLEL_MAKE_ARG=-j4
       - name: Build CI container
         uses: docker/build-push-action@v5
         with:
```

(and 3 more places)

Reply:
I have attempted to do this, but I'm not sure I got all the right places. Could you please check the new commit?